src/fields/Fields.cpp

```diff
@@ -389,49 +375,34 @@ LinCombination (const amrex::IntVect box_grow, amrex::MultiFab dst,
         const Array2<amrex::Real> dst_array = dst.array(mfi);
         const auto src_a_array = to_array2(src_a.array(mfi));
         const auto src_b_array = to_array2(src_b.array(mfi));
-        const amrex::Box bx = mfi.growntilebox(box_grow);
-        const int box_i_lo = bx.smallEnd(Direction::x);
-        const int box_j_lo = bx.smallEnd(Direction::y);
-        const int box_i_hi = bx.bigEnd(Direction::x);
-        const int box_j_hi = bx.bigEnd(Direction::y);
         amrex::ParallelFor(mfi.growntilebox(),
             [=] AMREX_GPU_DEVICE(int i, int j, int) noexcept
             {
-                const bool inside = box_i_lo<=i && i<=box_i_hi && box_j_lo<=j && j<=box_j_hi;
-                dst_array(i,j) =
-                    inside ? factor_a * src_a_array(i,j) + factor_b * src_b_array(i,j) : 0._rt;
+                dst_array(i,j) = factor_a * src_a_array(i,j) + factor_b * src_b_array(i,j);
             });
     }
 }
 
 /** \brief Calculates dst = factor*src. src can be a derivative
  *
- * \param[in] box_grow how much the domain of dst should be grown
  * \param[in] dst destination
  * \param[in] factor factor before src_a
  * \param[in] src first source
  */
 template<class FV>
 void
-Multiply (const amrex::IntVect box_grow, amrex::MultiFab dst,
-          const amrex::Real factor, const FV& src)
+Multiply (amrex::MultiFab dst, const amrex::Real factor, const FV& src)
```

Comment:
Previously the box growing was only used by extended_solve which was removed in this PR since its now possible to shrink the domain for particles. The field solvers now all use the valid domain without ghost cells so the number of cells is equal to amr.n_cell.

Reply:
Clear, and much better, thanks!